src/content/structured/components/cards/code.mdx

```diff
@@ -375,18 +549,55 @@ export const midImage = [
   subheading="Card subheading" 
   message="This is an example of a card with a middle image."
   style="width:360px;">
-  <svg slot="icon">
-    ...
-  </svg>  
+ <svg
+    slot="icon"
+    xmlns="http://www.w3.org/2000/svg"
+    height="24px"
+    viewBox="0 0 24 24"
+    width="24px"
+    fill="#000000"
+  >
+    <path d="M0 0h24v24H0V0z" fill="none" />
+    <path d="M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z" />
+  </svg>
   <ic-button variant="icon" title="More information" slot="interaction-button">
-    <svg slot="icon">
-      ...
-    </svg>
+    <svg
+        slot="icon"
+        xmlns="http://www.w3.org/2000/svg"
+        width="16"
+        height="16"
+        fill="currentColor"
+        class="bi bi-three-dots-vertical"
+        viewBox="0 0 16 16"
+      >
+        <path d="M9.5 13a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0zm0-5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0zm0-5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0z" />
+      </svg>
```

Comment:
Outdent so it lines up correctly (i.e. move back one tab space so closing svg lines up with opening) along with the SVG attributes